tswow-core/Private/TSItemTemplate.cpp

```diff
@@ -182,6 +234,190 @@ TSItemTemplate GetItemTemplate(uint32 entry)
+        << "," << info->Spells[0].SpellId << "," << info->Spells[0].SpellTrigger << "," << info->Spells[0].SpellCharges << "," << info->Spells[0].SpellPPMRate << "," << info->Spells[0].SpellCooldown << "," << info->Spells[0].SpellCategory << "," << info->Spells[0].SpellCategoryCooldown
+        << "," << info->Spells[1].SpellId << "," << info->Spells[1].SpellTrigger << "," << info->Spells[1].SpellCharges << "," << info->Spells[1].SpellPPMRate << "," << info->Spells[1].SpellCooldown << "," << info->Spells[1].SpellCategory << "," << info->Spells[1].SpellCategoryCooldown
+        << "," << info->Spells[2].SpellId << "," << info->Spells[2].SpellTrigger << "," << info->Spells[2].SpellCharges << "," << info->Spells[2].SpellPPMRate << "," << info->Spells[2].SpellCooldown << "," << info->Spells[2].SpellCategory << "," << info->Spells[2].SpellCategoryCooldown
+        << "," << info->Spells[3].SpellId << "," << info->Spells[3].SpellTrigger << "," << info->Spells[3].SpellCharges << "," << info->Spells[3].SpellPPMRate << "," << info->Spells[3].SpellCooldown << "," << info->Spells[3].SpellCategory << "," << info->Spells[3].SpellCategoryCooldown
+        << "," << info->Spells[4].SpellId << "," << info->Spells[4].SpellTrigger << "," << info->Spells[4].SpellCharges << "," << info->Spells[4].SpellPPMRate << "," << info->Spells[4].SpellCooldown << "," << info->Spells[4].SpellCategory << "," << info->Spells[4].SpellCategoryCooldown
+        << "," << info->Bonding << ",'" << info->Description << "'," << info->PageText << "," << info->LanguageID << "," << info->PageMaterial << "," << info->StartQuest
+        << "," << info->LockID << "," << info->Material << "," << info->Sheath << "," << info->RandomProperty << "," << info->RandomSuffix << "," << info->Block
+        << "," << info->ItemSet << "," << info->MaxDurability << "," << info->Area << "," << info->Map << "," << info->BagFamily << "," << info->TotemCategory
+        << "," << info->Socket[0].Color << "," << info->Socket[0].Content
+        << "," << info->Socket[1].Color << "," << info->Socket[1].Content
+        << "," << info->Socket[2].Color << "," << info->Socket[2].Content
+        << "," << info->socketBonus << "," << info->GemProperties << "," << int32(info->RequiredDisenchantSkill) << "," << info->ArmorDamageModifier << "," << info->Duration
+        << "," << info->ItemLimitCategory << "," << info->HolidayId << "," << info->ScriptId << "," << info->DisenchantID << "," << info->FoodType << "," << info->MinMoneyLoot
+        << "," << info->MaxMoneyLoot << "," << info->FlagsCu << ")";
+
+    //TC_LOG_ERROR("sql.sql", "query: `%s`", oss.str().c_str());
+    #if TRINITY
+    CharacterDatabase.PExecute("%s", oss.str().c_str());
+    #elif AZEROTHCORE
+    //CharacterDatabase.Execute(oss.str().c_str());//maybe? 
+    TS_LOG_ERROR("tswow.api", "TSItemTemplate::SaveItemTemplate not implemented for AzerothCore");
+    #endif
+
+}
```

Comment:
Yeah, this was a quick put together as I think I want to remove some parts of this query as I don't think every field is required to be editable.

Reply:
Also not sure if all parts are directly editable, from what I've tested they are but I haven't tested everything and I'm sure there must be some client limitations for some edits, just haven't come across them yet.

Reply:
i think that's good to verify for something like this so we don't publish stuff we have to remove. Of course, it can stay for now until someone wants to look into it

Reply:
yeah, moving it to a prepared statement won't be hard once all of the parts are verified that they work. like I'm unsure if onUse or passive spells would effect with just a reload like this.

Reply:
all fields are added but not all tested, waiting to move to prepared statements until all fields tested